sdk/txbuildtest/txbuildtest.go

```diff
@@ -57,3 +58,78 @@ func BuildResultMetaXDR(ledgerEntryResults []xdr.LedgerEntryData) (string, error
 	}
 	return tmXDR, nil
 }
+
+type FormationResultMetaParams struct {
+	InitiatorSigner string
+	ResponderSigner string
+	InitiatorEscrow string
+	ResponderEscrow string
+	StartSequence   int64
+	Asset           txnbuild.Asset
+}
+
+func BuildFormationResultMetaXDR(params FormationResultMetaParams) (string, error) {
```

Comment:
added this helper method. When generating the resultMetaXDR for the integration tests this chunk of code for the formationTx Result XDR was written enough and took up annoyingly enough space that it was worth abstracting away